src/main/java/com/checkout/payments/links/PaymentLinksClientImpl.java

```diff
@@ -11,16 +12,16 @@
     public static final String PAYMENT_LINKS = "/payment-links";
 
     public PaymentLinksClientImpl(final ApiClient apiClient, final CheckoutConfiguration configuration) {
-        super(apiClient, configuration);
+        super(apiClient, new SecretKeyCredentials(configuration));
```

Comment:
Please use `SecrectKeyCredentials.fromConfiguration(configuration)` instead. Public constructor is deprecated and it will be made private in future versions. Please this change in all `SecretKeyCredentials` created in this PR.